src/modules/QtQuick/Flow.js

```diff
@@ -55,9 +61,12 @@ registerQmlType({
             curVPos += child.height + this.spacing;
         }
     }
-    if (this.flow == 0)
-        this.implicitHeight = curVPos + rowSize;
-    else
-        this.implicitWidth = curHPos + rowSize;
+    if (this.flow === Flow.LeftToRight) {
```

Comment:
See https://github.com/qmlweb/qmlweb/commit/032f73242906e6a19edff1189367ebbce52e6d30, for example.